bin/smokerd.py

```diff
@@ -12,16 +12,30 @@
 
 def main():
     parser = argparse.ArgumentParser(description='Smoke testing daemon')
```

Comment:
It would be better to use `formatter_class=argparse.ArgumentDefaultsHelpFormatter` to always print default values and not describing them in help for each parameter.